<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notas</class>
 <widget class="QDialog" name="notas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notas de la empresa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="texpushButton">
       <property name="text">
        <string>&amp;Tex</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/tex.png</normaloff>:/iconos/graf/tex.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimirpushButton">
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton">
       <property name="text">
        <string>C&amp;opiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>texpushButton</tabstop>
  <tabstop>imprimirpushButton</tabstop>
  <tabstop>copiarpushButton</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>aceptarpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>notas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
